tests/test_utils.py

```diff
@@ -386,6 +386,30 @@ def test_get_compile_command(tmpdir_cwd, cli_args, expected_command):
         assert get_compile_command(ctx) == expected_command
 
 
+@pytest.mark.parametrize(
+    ("config_file", "expected_command"),
+    (
+        pytest.param(
+            "pyproject.toml", "pip-compile", id="exclude default pyproject.toml"
```

Comment:
Does pip-sync produce these headers too? I'd add it to the tests, then.

Reply:
Nope, pip-sync does not use `get_compile_command`.

Reply:
Marking as resolved.